### ISSUE TITLE
Scrollbar occludes buttons

### ISSUE THREAD

**Issue by xactlyblue** [flipbook-labs/flipbook#213]
## Describe the bug
The scrollbar slightly occludes the buttons
## Steps to reproduce

1.
2.
3.

**Comment by passhley** [comment]
yeah, this is something that will just be there until I manage the time to finish the new ui.
unfortunately, free time is something i don't have a lot of 😭
